### DISCUSSION TITLE
好耶！领导终于把绳网账号给我运营了！

### DISCUSSION THREAD

**Discussion by SSTV-10068** [share121/inter-knot#923] [其他]
呃啊啊啊啊（来着某00后小伙的哀嚎），以后不知道怎么运营怎么办？！！
![20230317144016_46374](https://github.com/user-attachments/assets/d55c5b1d-41a9-4b85-986a-e8a4fe9efa36)
![v2-97c3651afeee1a88d6d515fffc04b3a5_b](https://github.com/user-attachments/assets/4c520997-2004-4b3d-b2ee-721f516458d0)

**Comment by ssj20051003** [comment]
啊啊，猫meme不要入侵绳网！

**Comment by BOXANK** [comment]
我嘞个豆

**Comment by warning-Kirov-reporting** [comment]
或许可以在这多分享一些猫meme，大家会喜欢的。

**Comment by yesanbai** [comment]
流量公式也来绳网了，爱了爱了。

**Comment by tyhpoi** [comment]
猫猫meme也是来绳网了

**Comment by gaojiaobei** [comment]
我勒个官号猫meme啊

**Comment by SSTV-10068** [comment]
我错了，我再也不发猫meme了

**Comment by Boring1998** [comment]
怎么会

**Reply by BJ-cmyk** [reply]
怎么会...💀

**Reply by Yizakl** [reply]
怎么会↗️怎么会↘️

**Reply by Yizakl** [reply]
哦，多么美的一颗心，
怎么会？怎么会？
就成为了一滩烂泥
哦，多么简单的一首诗
怎么会？怎么会？
就充满了讽刺
我想要说的前人们都说过了
我想要做的有钱人都做过了
我想要的公平是一切不公的虚构的

**Comment by ParanoidHikari** [comment]
猫meme入侵绳网

**Comment by manyshoes** [comment]
公式化猫meme，就得配公式化评论啊

**Comment by LiuLiBHZ** [comment]
现在做毛meme不如做大头小马的视频，曼波几句一定爆火。
